docs/faq/questions/cloud-faq.mdx

```diff
@@ -261,5 +261,8 @@ Cloud.
 you may also need to add both the following to the version control IP allow
```

Comment:
"both" is no longer correct if there are no longer just 2 IP addresses to consider.
```suggestion
you may also need to add the following to the version control IP allow
```

Reply:
Good catch, Thank you Mike